koku/sources/sources_error_message.py

```diff
@@ -62,7 +41,6 @@ def general_errors(self, message):
     def _display_string_function(self, key):
         """Return function to get user facing string."""
         ui_function_map = {
-            ProviderErrors.AZURE_CLIENT_ERROR: self.azure_client_errors,
```

Comment:
Are you sure that the error message is being sent back to the user? I feel like now the string function will always be None:
```
string_function = ui_function_map.get(key)
        return string_function
```

Reply:
we fallback to the internal message if no display function is found

```
display_fn = self._display_string_function(key)
if display_fn:
    ...
else:
    display_message = internal_message

```

Reply:
I completely missed that David 👍 Thanks for the additional clarity there. 🙇